Upload A File/Download Upload Library.gcomp/javascript/UploadFileToWebVI.js

```diff
@@ -12,46 +12,96 @@
 // http://jsfiddle.net/VBJ9h/319/
 // https://github.com/ni/webvi-examples
 // https://developer.mozilla.org/en-US/docs/Web/API/Window
-
-var fileText = "";
-var fileRead = false;
-var inputElementName = "myFile";
```

Comment:
Another thing to be aware of is that because these variables are being shared between the other functions, The `Upload A File.gvi` has to have reentrancy disabled (which it already does). If reentrancy was enabled and this VI was allowed to be called in parallel then the calls would be overwriting the same shared variables.

While not a big deal in this case because it's not really useful to have multiple parallel file dialogs opening at once (might even be annoying) it's important to consider for other apis (ie waiting for responses on parallel websocket or bluetooth connections, etc).